serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/client/brreg/BrregClientImpl.java

```diff
@@ -53,7 +53,7 @@ public BrregClientImpl(URI uri) {
         try {
             enhet= rt.getForObject(currentURI, BrregEnhet.class);
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)